### PR TITLE
#170574394 user should be able to verify email address

### PR DESCRIPTION
**Pull request by victkarangwa**
#### What does this PR do?
This PR adds email verification page after user signs up
#### Description of Task to be completed?
- A user is able to resend a verification email after signup
- A user is able to navigate directly to the app after email verification
- User is able to get a successful registration page
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-frontend`
- `git checkout ft-user-verification-email-170574394`
- `npm install` packages
- `npm run dev` to start the development server
- It should open your default browser automatically
- Navigate to `localhost:4000/verify-email` to access the page directly
#### Any background context you want to provide?
Previously, the users used to validate their email using the backend part
#### What are the relevant pivotal tracker stories?
[#170574394](https://www.pivotaltracker.com/story/show/170574394)
###### verify email page
Screen Shot 2020-01-15 at 09 35 46

###### Successful email verication page
Screen Shot 2020-01-10 at 02 01 13

#### Questions:
N/A

### CODE REVIEW

File: src/containers/user/resendEmail.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        <div data-testid="resendEmail-Container" className="boxContainer">
+          <div className="fieldContainer">
+            <div className="center">
+              <img
+                className="square-image"
+                alt="barefoot_logo"
+                src={`
+   https://res.cloudinary.com/victorkarangwa4/image/upload/v1578477999/icons8-email_open_filled_n80pwf.png`}
+              />
+            </div>
```
Comment (william000000) [inline src/containers/user/resendEmail.js]: It should be better when this cloudinary link comes from `.env`  instead of showing up the whole link.

**Review by william000000 — CHANGES_REQUESTED** [review]

**Review by hezronkimutai — APPROVED** [review]
Nice work @victorkarangwa4

**Review by NiyongaboEric — CHANGES_REQUESTED** [review]
Well done @victorkarangwa4. All test are passing and UI is responsive. However, I am not able to run your work locally.
Screen Shot 2020-01-16 at 07 08 06

**Review by NiyongaboEric — APPROVED** [review]

**Review by william000000 — APPROVED** [review]
@victorkarangwa4  Looks good to go!

**Review by shemaeric — APPROVED** [review]
Good work @victorkarangwa4

**Pull request merged**